ansible/install-apps.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_file: /etc/apt/sources.list.d/google-chrome.list
+  - apt_file_Git: 
+  tasks:
+  - name: Does the Google apt file exist?
+    command: test -f {{apt_file}}
```

Comment:
So after implementing comments on the next task, this one won't be necessary anymore.